src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -192,9 +210,10 @@ object DockerPlugin extends AutoPlugin {
 
       stage0 ++ stage1
     }
-  ) ++ mapGenericFilesToDocker ++ inConfig(Docker)(
+  ) ++ inConfig(Docker)(
     Seq(
       executableScriptName := executableScriptName.value,
+      mappings := mappingsInDocker((mappings in Universal).value, defaultLinuxInstallLocation.value),
```

Comment:
Since `Docker / mappings` haven't changed, we should probably roll back these changes when cleaning up, just to keep the PR focused on one thing.

Reply:
Makes sense, but later in separate PR I'd implement this change as previous method is used in uncommon way.